### PR TITLE
fix 2.0 api docs

### PR DESCRIPTION
**Pull request by zhupengyang**
- standard_normal 
standard_normal

- hsigmoid_loss
hsigmoid_loss

- normal
normal

- prelu fluid
prelu_fluid

- LogSoftmax
LogSoftmax

- Hardtanh
Hardtanh

- Hardsigmoid
Hardsigmoid

- Hardshrink
Hardshrink

- log_softmax
log_softmax

- leaky_relu
leaky_relu

- gelu
gelu

- elu
elu

- zeros_like
zeros_like

- var
var

- uniform
uniform

- std
std

- randperm
randperm

- randn
randn

- randint
randint

- rand
rand

- ones_like
ones_like

- mean
mean

- arange
arange

### CODE REVIEW

**Review by hong19860320 — APPROVED** [review]
LGTM

File: doc/paddle/api/paddle/fluid/layers/prelu_cn.rst [hunk 1]
```diff
@@ -3,16 +3,12 @@
 prelu
 -------------------------------
 
-
 .. py:function:: paddle.fluid.layers.prelu(x, mode, param_attr=None, name=None)
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/layers/prelu_cn.rst]: 这里最好改下啊，paddle.static.nn.prelu

File: doc/paddle/api/paddle/fluid/layers/prelu_cn.rst [hunk 2]
```diff
@@ -24,27 +20,23 @@ prelu
...
-          - **mode** (str) - 权重共享模式。
-          - **param_attr** (ParamAttr，可选) - 可学习权重 :math:`[\alpha]` 的参数属性，可由ParamAttr创建。默认值为None，表示使用默认的权重参数属性。具体用法请参见 :ref:`cn_api_fluid_ParamAttr` 。
-          - **name** (str，可选) – 具体用法请参见 :ref:`api_guide_Name` ，一般无需设置，默认值为None。 
+    - **x** （Tensor）- 多维Tensor或LoDTensor，数据类型为float32。
+    - **mode** (str) - 权重共享模式。
+    - **param_attr** (ParamAttr，可选) - 可学习权重 :math:`[\alpha]` 的参数属性，可由ParamAttr创建。默认值为None，表示使用默认的权重参数属性。具体用法请参见 :ref:`cn_api_fluid_ParamAttr` 。
+    - **name** (str，可选) – 具体用法请参见 :ref:`api_guide_Name` ，一般无需设置，默认值为None。 
 
 
 返回： 表示激活输出Tensor或LoDTensor，数据类型为float32。与输入形状相同。
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/layers/prelu_cn.rst]: 不用暴露LoDTensor吧

File: doc/paddle/api/paddle/tensor/creation/ones_like_cn.rst [hunk 3]
```diff
@@ -28,10 +28,7 @@ ones_like
...
     import paddle
-    import numpy as np
 
-    paddle.enable_imperative()
-
-    x = paddle.imperative.to_variable(np.array([1,2,3], dtype='float32'))
-    out1 = paddle.ones_like(x) # [1., 1., 1.]
-    out2 = paddle.ones_like(x, dtype='int32') # [1, 1, 1]
+    x = paddle.to_tensor([1,2,3])
+    out1 = paddle.zeros_like(x) # [1., 1., 1.]
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/tensor/creation/ones_like_cn.rst]: paddle.ones_like

File: doc/paddle/api/paddle/tensor/creation/ones_like_cn.rst [hunk 4]
```diff
@@ -28,10 +28,7 @@ ones_like
...
+    out2 = paddle.zeros_like(x, dtype='int32') # [1, 1, 1]
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/tensor/creation/ones_like_cn.rst]: paddle.ones_like

File: doc/paddle/api/paddle/fluid/layers/prelu_cn.rst (hunk 1 above)
Comment (zhupengyang) [reply doc/paddle/api/paddle/fluid/layers/prelu_cn.rst]: Done

File: doc/paddle/api/paddle/fluid/layers/prelu_cn.rst (hunk 2 above)
Comment (zhupengyang) [reply doc/paddle/api/paddle/fluid/layers/prelu_cn.rst]: Done

File: doc/paddle/api/paddle/tensor/creation/ones_like_cn.rst (hunk 3 above)
Comment (zhupengyang) [reply doc/paddle/api/paddle/tensor/creation/ones_like_cn.rst]: Done

File: doc/paddle/api/paddle/tensor/creation/ones_like_cn.rst (hunk 4 above)
Comment (zhupengyang) [reply doc/paddle/api/paddle/tensor/creation/ones_like_cn.rst]: Done

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**